README.md

```diff
@@ -1,26 +1,31 @@
 <h1>
   <img src="./_doc/logo.svg" align="left" height="46px" alt="Consul logo"/>
-  <span>Consul Dataplane<sup>BETA</sup></span>
+  <span>Consul Dataplane</span>
 </h1>
 
 Consul Dataplane is a lightweight process that manages Envoy for Consul service mesh workloads.
 
 Consul Dataplane's design removes the need to run Consul client agents. Removing Consul client
 agents results in the following benefits:
 
-- **Fewer network requirements**: Consul client agents use multiple network protocols and require
-  bi-directional communication between Consul client and server agents for the gossip protocol.
-  Consul Dataplane does not use gossip and instead uses a single gRPC connection out to the Consul servers.
-- **Simplified set up**: Consul Dataplane does not need to be configured with a gossip encryption key
-  and operators do not need to distribute separate ACL tokens for Consul client agents.
-- **Additional runtime support**: Consul Dataplane runs as a sidecar alongside your workload, making
-  it easier to support various runtimes. For example, it runs on serverless platforms when you do not have access
-  to the host machine where the Consul client agent can be run.
-- **Easier upgrades**: Deploying new Consul versions no longer requires  upgrading Consul
-  client agents. Consul Dataplane has better compatibility across Consul server versions, so
-  you only need to upgrade the Consul servers to take advantage of new Consul features.
-
-Refer to the [Documentation](#documentation) section for more information on Consul Dataplane.
+- **Fewer networking requirements**: Without client agents, Consul does not require bidirectional
+  network connectivity across multiple protocols to enable gossip communication. Instead, it
+  requires a single gRPC connection to the Consul servers, which significantly simplifies
+  requirements for the operator.
+- **Simplified set up**: Because there are no client agents to engage in gossip, you do not have to
+  generate and distribute a gossip encryption key to agents during the initial bootstrapping
+  process. Securing agent communication also becomes simpler, with fewer tokens to track,
+  distribute, and rotate.
+- **Additional environment and runtime support**: Current Consul on Kubernetes deployments require
+  using hostPorts and DaemonSets for client agents, which limits Consul’s ability to be deployed in
+  environments where those features are not supported. As a result, Consul Dataplane supports AWS
+  Fargate and GKE Autopilot.
+- **Easier upgrades**: With Consul Dataplane, updating Consul to a new version no longer requires
+  upgrading client agents. Consul Dataplane also has better compatibility across Consul server
+  versions, so the process to upgrade Consul servers becomes easier.
+
+Refer to the [Documentation](https://developer.hashicorp.com/consul/docs/connect/dataplane) for more
```

Comment:
I updated this link to the docs website, and remove the rest of the inline documentation here? No need to duplicate the doc content.